AnnoyancesFilter/sections/tweaks.txt

```diff
@@ -1389,7 +1388,12 @@ nekopoi.web.id#%#AG_onLoad(function() { document.body.oncontextmenu = document.b
 terbit21.online,anisubindo.video,ustv247.tv#%#AG_onLoad(function() { document.ondragstart = document.oncontextmenu = function() { return true; }; });
 kawanfilm21.pro#%#AG_onLoad(function() { document.onmousedown = document.onmouseup = document.onselectstart = function() { return true; }; });
 tistory.com,dinvemitoloji.com#%#AG_onLoad(function() { document.ondragstart = document.onselectstart = document.oncontextmenu = function() { return true; }; });
-poisonous-raspberry-fields.blogspot.com#%#AG_onLoad(function(){var a=document.getElementsByTagName("body")[0];a&&(a.removeAttribute("onmousedown"),a.removeAttribute("onselectstart"),a.removeAttribute("ondragstart"));a=document.querySelectorAll(".post-body[ondragstart][onmousedown][onselectstart]");for(var b=0;b<a.length;b++)a[b].removeAttribute("onmousedown"),a[b].removeAttribute("onselectstart"),a[b].removeAttribute("ondragstart")});
+poisonous-raspberry-fields.blogspot.com#%#//scriptlet("remove-attr", "onmousedown", "body")
+poisonous-raspberry-fields.blogspot.com#%#//scriptlet("remove-attr", "onselectstart", "body")
+poisonous-raspberry-fields.blogspot.com#%#//scriptlet("remove-attr", "ondragstart", "body")
+poisonous-raspberry-fields.blogspot.com#%#//scriptlet("remove-attr", "onmousedown", ".post-body[ondragstart][onmousedown][onselectstart]")
+poisonous-raspberry-fields.blogspot.com#%#//scriptlet("remove-attr", "onselectstart", ".post-body[ondragstart][onmousedown][onselectstart]")
+poisonous-raspberry-fields.blogspot.com#%#//scriptlet("remove-attr", "ondragstart", ".post-body[ondragstart][onmousedown][onselectstart]")
```

Comment:
`poisonous-raspberry-fields.blogspot.com#%#//scriptlet("remove-attr", "onmousedown|onselectstart|ondragstart", "body, .post-body[ondragstart][onmousedown][onselectstart]")`. For the similar rules do the same thing.